Documentation/ExtensionArchitecture/Tutorials/Tea/Model.rst

```diff
@@ -0,0 +1,298 @@
+
+The TCA makes the database model known to TYPO3. It defines all fields
+containing data and all semantic fields that have a special meaning within
+TYPO3 (like the :sql:`deleted` field used for soft deleting).
+
+TCA also defines how the input fields for the data in the backend should look
+like.
+
+The TCA is a nested PHP array. On the first level we need the the following
+keys in this example:
+
+:php:`ctrl`
+    Settings for the complete table, like an overall title, a lable that
+    should specify the single object, default sorting and the names of some
+    fields used internally.
+
+:php:`types`
+    We only have one type of tea record, however it is mandatory to
+    describe at least one type. Here we define the order in which
+    the fields are displayed in the backend.
+
+:php:`columns`
+    Here we define all fields that can be used for user input in the
+    backend.
+
+
+..  _extbase_tutorial_tea_model_ctrl:
+
+TCA :php:`ctrl` - Settings for the complete table
+-------------------------------------------------
+
+..  include:: /CodeSnippets/Tutorials/Tea/Configuration/TCA/TeaCtrl.rst.txt
+
+..  _extbase_tutorial_tea_model_ctrl_title:
+
+:php:`title`
+~~~~~~~~~~~~
+
+Defines the title used when we are talking about the table in the backend.
+It will be displayed on top of the list view of records in the backend
+and in backend forms.
+
+..  figure:: /Images/ManualScreenshots/ExtensionArchitecture/Tutorials/Tea/TeaTitle.png
+    :class: with-shadow
+
+    The **title** of the table tea
+
+Strings starting with :php:`LLL:` will be localized. See chapter
+:ref:`Extension localization <extension_localization>`. All other strings
+would be output as strings. You could also write:
+
+.. code-block:: php
+   :caption: EXT:tea/Configuration/TCA/tx_tea_domain_model_product_tea.php
+
+   [
+       'ctrl' => [
+           'title' => 'Tea',
+       ],
+   ]
+
+..  _extbase_tutorial_tea_model_ctrl_label:
+
+:php:`label`
+~~~~~~~~~~~~
+
+The :php:`label` is used as name for a specific tea record. The name is used
+in listings, dropdown fields used to reference a tea record and in the forms
+for editing records in the backend:
+
+
+..  figure:: /Images/ManualScreenshots/ExtensionArchitecture/Tutorials/Tea/TeaLabel.png
+    :class: with-shadow
+
+    The **label** of a tea record. Yeah, sorry, Coffee is my tea...
+
+..  _extbase_tutorial_tea_model_ctrl_others:
+
+:php:`tstamp`, :php:`deleted`, ...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+These fields are used for different aspects of handling tables. You can read in
+depth in the `TCA Reference, chapter ctrl <t3tca:ctrl>` about what they mean and
+are used for.
+
+..  _extbase_tutorial_tea_model_columns:
+
+TCA :php:`columns` - Defining the database fields
+-------------------------------------------------
+
+All fields that can be changed in the TYPO3 backend or used in the Extbase
+model have to be listed here. Otherwise they will not be recognized by
+any API call to the table.
```

Comment:
There are APIs that don't care about the TCA, but TYPO3's backend won't recognize and display the fields unless you mention them here.

```suggestion
model have to be listed here. Otherwise they will not be recognized by TYPO3.
```